### PR TITLE
Updates from audit

### PR DESCRIPTION
**Pull request by sofiahaglund**
- Changed "validated" to "accredited"
- Added disclaimer about deviations (which will be manually updated when this is relevant)
- Added disclaimer about user-provided elements

### CODE REVIEW

File: data/report_templates/project_summary.md [hunk 1]
```diff
@@ -154,6 +158,10 @@ NGI ID | User ID | Status
 
 In cases where samples and/or libraries have failed the QC, the user is always consulted regarding how to proceed. If the user wishes to proceed to sequence the failed samples and/or libraries, NGI bears no responsibility regarding the quality and number of reads of the sequenced sample data.
 
+## User-provided samples, information and data
+
+The NGI does not take responsibility for how user-provided samples, information, or data may affect sequencing results.
```
Comment (aanil) [inline data/report_templates/project_summary.md:163]: Should it be 'The' NGI or just NGI? The rest of the document doesn't use the definite version.

Comment (sofiahaglund) [reply data/report_templates/project_summary.md:163]: I asked Phil to look at it, and he specifically wanted it to say "the" NGI :-D
Hm so either I would have to change the whole document or we remove the "the", maybe it's the easiest to just remove the "the", sorry Phil :-D

**Comment by parlundin** [comment]
Looks fine for me but waiting for 2nd opinion from QA-group

File: data/report_templates/project_summary.md (hunk 1 above)
Comment (ewels) [reply data/report_templates/project_summary.md]: Remove it, I have been outvoted on this many times. I will keep fighting, but I have lost this battle ☝🏻 😆

**Review by aanil — APPROVED** [review]
👍  Looks good

**Comment by sofiahaglund** [comment]
OK, there will be more updates, so no merging just yet! :-D

**Comment by aanil** [comment]
Are there going to be more updates to this? @sofiahaglund

**Review by parlundin — APPROVED** [review]
All looks fine, QA aproved

**Pull request merged**